core/filter_constants_inc.php

```diff
@@ -67,3 +77,13 @@
 define( 'FILTER_PROPERTY_PLATFORM', 'platform' );
 define( 'FILTER_PROPERTY_OS', 'os' );
 define( 'FILTER_PROPERTY_OS_BUILD', 'os_build' );
+
+# These are old date fields names, keeping them for backward compatibility, if used externally
+define( 'FILTER_PROPERTY_START_DAY', FILTER_PROPERTY_DATE_SUBMITTED_START_DAY );
```

Comment:
The PR implements also the ability to filter by last updated date https://www.mantisbt.org/bugs/view.php?id=6823
So we have two times a START_DAY.
Not sure if this is reason enough to rename the old constants, but it's a bit more consistent.

@cproensa wouldn't it be easier to understand if the new constants contain also "DATE", e.g. FILTER_PROPERTY_DATE_LAST_UPDATED_START_DAY